spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Utils.scala

```diff
@@ -570,4 +570,57 @@ object Utils {
     times: Array[(AbstractModule[_ <: Activity, _ <: Activity, T], Long, Long)]): (Long, Long) = {
       times.map(t => (t._2, t._3)).reduce((a, b) => (a._1 + b._1, a._2 + b._2))
   }
+
+  private[nn] def calc(tensor: Tensor[Float], mask: Int): Array[Float] = {
```

Comment:
I would suggest to add a comment on top of method header.
Whats the representation of the mask, decimal? binary?; if binary, its Big endian or little endian?